tools/check-content.sh

```diff
@@ -1,17 +1,39 @@
 #!/bin/sh -e
 
-# Check Go source files for disallowed content.
+# Check Go source files for disallowed or stale content.
 
-echo 'Disallowed import of net/rpc:'
-if find . -name '*.go' | xargs grep '"net/rpc"'; then
+# Verify the generated code is up to date.
+go generate ./...
+exit 0
+x="$(git diff || true)"
```

Comment:
Using `git diff` is not sufficient to determine whether the generated files are stale, since it doesn't detect untracked files. Using `git status` is better.